app/models/article.rb

```diff
@@ -4,4 +4,32 @@ class Article < ApplicationRecord
   def title?
     title.present?
   end
+
+  # Test Rails Best Practices warning (remove unused methods)
+  def unused_method
+    has_title
+    too_many
+  end
+
+  # Test Rubocop warning (Naming/PredicateName)
+  def has_title
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/sample-rails-ruby/blob/1cfea6674ee2689634e2a2b7590162d2a5f33cbd/app/models/article.rb#L15"></span>Naming/PredicateName: Rename <code>has_title</code> to <code>title?</code>. (https://github.com/rubocop-hq/ruby-style-guide#bool-methods-qmark)</td>
    </tr>
  </tbody>
</table>